api/tests/test_email_address_scalar.py

```diff
@@ -15,7 +15,8 @@
     os.path.realpath(os.path.join(os.getcwd(), os.path.expanduser(__file__))))
 sys.path.append(os.path.normpath(os.path.join(SCRIPT_DIR, PACKAGE_PARENT)))
 
-from scalars.email_address import EmailAddress, scalar_error_type, scalar_error_only_types
+from scalars.email_address import EmailAddress, scalar_error_type, \
```

Comment:
Should make this consistent with rest of project and use `from <package> import (<i1>, <i2>, ... , <in>)` form.  Just add the parentheses.